git/internal/e2e/go.mod

```diff
@@ -0,0 +1,124 @@
+module github.com/fluxcd/pkg/git/internal/e2e
```

Comment:
Wondering if we need to make this internal.
Usually the tests are placed in a `tests` directory with more subdirectories for the types of tests.
Internal is used to make packages private. But from what I've seen, the tests directory is not made internal and this package seems to be in a similar situation.

Reply:
Yes let's make the tests internal please.

Reply:
I decided to make it internal because it has it's own `go.mod`. Otherwise, I would agree with you @darkowlzz